cli/cmd/generate_azure.go

```diff
@@ -0,0 +1,680 @@
+				// we are going to write out new cached values, so if they run it - bail out - and run it again they'll get
+				// re-prompted.
+				if answer {
+					// Merge cached inputs to current options (current options win)
+					if err := mergo.Merge(GenerateAzureCommandState, cachedOptions); err != nil {
+						return errors.Wrap(err, "failed to load saved options")
+					}
+					if err := mergo.Merge(GenerateAzureCommandExtraState, extraState); err != nil {
+						return errors.Wrap(err, "failed to load saved options")
+					}
+				}
+			}
+
+			// Collect and/or confirm parameters
+			err = promptAzureGenerate(GenerateAzureCommandState, GenerateAzureCommandExtraState)
+			if err != nil {
+				return errors.Wrap(err, "collecting/confirming parameters")
+			}
+
+			return nil
+		},
+	}
+)
+
+func validateStorageLocation(location string) error {
+	if !validStorageLocations[location] {
+		return errors.New("invalid storage location supplied")
+	}
+	return nil
+}
+
+func initGenerateAzureTfCommandFlags() {
+	// Azure sub-command flags
+	generateAzureTfCommand.PersistentFlags().BoolVar(
+		&GenerateAzureCommandState.ActivityLog, "activity_log", false, "enable active log integration")
+
+	generateAzureTfCommand.PersistentFlags().StringVar(
+		&GenerateAzureCommandState.ActivityLogIntegrationName,
+		"activity_name",
+		"",
+		"specify name of activity log, if none given will default to TF activity log")
+
+	generateAzureTfCommand.PersistentFlags().BoolVar(
+		&GenerateAzureCommandState.Config, "config", false, "enable config integration")
+
+	generateAzureTfCommand.PersistentFlags().StringVar(
+		&GenerateAzureCommandState.ConfigIntegrationName,
+		"config_name",
+		"",
+		"specify name of config, if none given will default to TF config")
+
+	generateAzureTfCommand.PersistentFlags().BoolVar(
+		&GenerateAzureCommandState.CreateAdIntegration, "ad_create", true, "create new active directory integration")
+	generateAzureTfCommand.PersistentFlags().BoolVar(
+		&GenerateAzureCommandState.ManagementGroup, "mgmt_group", false, "use management rather than subscriptions")
+	generateAzureTfCommand.PersistentFlags().StringVar(
+		&GenerateAzureCommandState.ManagementGroupId,
+		"mgmt_group_id",
+		"",
+		"specify managment group id")
+
+	generateAzureTfCommand.PersistentFlags().BoolVar(
+		&GenerateAzureCommandState.ExistingStorageAccount, "existing_storage", false, "use existing storage account")
+	generateAzureTfCommand.PersistentFlags().StringVar(
+		&GenerateAzureCommandState.StorageAccountName,
+		"strg_account_name",
+		"",
+		"specify storage account name")
+	generateAzureTfCommand.PersistentFlags().StringVar(
+		&GenerateAzureCommandState.StorageAccountResourceGroup,
+		"strg_res_group",
+		"",
+		"specify storage resource group")
+	generateAzureTfCommand.PersistentFlags().StringVar(
+		&GenerateAzureCommandState.StorageLocation,
+		"location",
+		"",
+		"specify azure region where storage account logging resides")
```

Comment:
The Description on the Terraform registry for this field is `The Azure Region in which all resources should be created`
I wonder if this could be changed to match the wording of the registry field description
```suggestion
		&GenerateAzureCommandState.StorageLocation,
		"location",
		"",
		"specify azure region in which all resources should be created")
```

Reply:
The definition here https://registry.terraform.io/modules/lacework/activity-log/azure/latest is exactly that, it relates to the location where the storage location logging resides

Reply:
oh nice. Not sure what version I was looking at when I made this comment

Reply:
Actually now I see it... We have a difference between the README and the inputs field description...
https://registry.terraform.io/modules/lacework/activity-log/azure/latest?tab=inputs

Reply:
Would you mind raising a PR for this module to update that input description?

Reply:
Will do